src/services/sos/Sos.cpp

```diff
@@ -71,8 +71,11 @@ void pack_snapshot(SOS_pub* sos_pub, int snapshot_id, const std::map< Attribute,
             ;
         }
     }
-
-    SOS_publish(sos_pub);
+    if (yn_publish == true) {
```

Comment:
Seems we don't call this with `yn_publish = true` anywhere. Do we need this option then?

Reply:
No we can trim that out for now. That was my initial attempt to correct the "publish with every pack" issue. There will never be a case where we want to publish for each element of a snapshot.

There may soon be cases where we want to register more than one triggering event, but for now this is totally sufficient and we'll code that in when we get there.